packages/react/src/components/Table/TablePropTypes.js

```diff
@@ -88,7 +88,7 @@ export const TableRowPropTypes = PropTypes.arrayOf(
 export const TableColumnsPropTypes = PropTypes.arrayOf(
   PropTypes.shape({
     id: PropTypes.string.isRequired,
-    name: PropTypes.string.isRequired,
+    name: PropTypes.node.isRequired,
```

Comment:
```suggestion
    name: PropTypes.oneOfType([PropTypes.node, PropTypes.string].isRequired,
```

Might be better to explicitly allow both strings and nodes still

Reply:
Done!